src/ui/buttons/styles/button.js

```diff
@@ -125,4 +125,27 @@ export const buttonStyle = `
     .${ CLASS.CONTAINER } .${ CLASS.VAULT_HEADER } {
         margin-top: 10px;
     }
+
+    .${ CLASS.BUTTON }.${ CLASS.COLOR }-${ BUTTON_COLOR.BLACK } {
+        color: ${ COLOR.BLACK };
+    }
+
+    .${ CLASS.BUTTON }.${ CLASS.COLOR }-${ BUTTON_COLOR.WHITE } {
+        color: ${ COLOR.WHITE };
+    }
+
+    .${ CLASS.BUTTON }[${ ATTRIBUTE.FUNDING_SOURCE }=${ FUNDING.CARD }] * {
+        height: 1.2em;
```

Comment:
Can we keep the logo size consistent with other buttons?

Reply:
Short answer, no. 
These are how the buttons look like without the `height` and `font-size` changes (with various viewport sizes)

large:
![sdk htm 2019-12-16 14-39-19](https://user-images.githubusercontent.com/3213579/70949116-d9de9f80-2011-11ea-94e2-43412e598c57.png)

medium:
![sdk htm 2019-12-16 14-38-46](https://user-images.githubusercontent.com/3213579/70949101-d0edce00-2011-11ea-9449-c98807ed200f.png)

small:
![sdk htm 2019-12-16 14-38-04](https://user-images.githubusercontent.com/3213579/70949057-bb78a400-2011-11ea-8e73-a8034610095b.png)

With the style adjustments 

large:
![sdk htm 2019-12-16 14-41-24](https://user-images.githubusercontent.com/3213579/70949244-2924d000-2012-11ea-9e77-ec6430172e44.png)

medium:
![sdk htm 2019-12-16 14-41-47](https://user-images.githubusercontent.com/3213579/70949258-3215a180-2012-11ea-9841-ef99910015a7.png)


small:
![sdk htm 2019-12-16 14-42-31](https://user-images.githubusercontent.com/3213579/70949309-4fe30680-2012-11ea-9a30-d135b661994d.png)